src/azure-cli/azure/cli/command_modules/iot/commands.py

```diff
@@ -59,13 +59,13 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         g.generic_update_command('update', getter_name='iot_dps_get', setter_name='iot_dps_update',
                                  command_type=update_custom_util)
 
-    # iot dps access-policy commands
-    with self.command_group('iot dps access-policy', client_factory=iot_service_provisioning_factory) as g:
-        g.custom_command('list', 'iot_dps_access_policy_list')
-        g.custom_show_command('show', 'iot_dps_access_policy_get')
-        g.custom_command('create', 'iot_dps_access_policy_create', supports_no_wait=True)
-        g.custom_command('update', 'iot_dps_access_policy_update', supports_no_wait=True)
-        g.custom_command('delete', 'iot_dps_access_policy_delete', supports_no_wait=True)
+    # iot dps access-policy commands (Deprecated)
+    with self.command_group('iot dps access-policy', client_factory=iot_service_provisioning_factory, deprecate_info=self.deprecate(redirect='iot dps policy', hide=True)) as g:
```

Comment:
we talked about explicitly leaving the deprecation visible for more user exposure and setting `expiration` a couple versions out.